### ISSUE TITLE
Publish PyNVML extension

### ISSUE THREAD

**Issue by mrocklin** [ian-r-rose/jupyterlab-bokeh-server#8]
In https://github.com/rjzamora/jupyterlab-bokeh-server/tree/pynvml @rjzamora has extended this project to support GPU diagnostics.  It would be nice to publish this as a JupyterLab extension so that people could more easily start using it. 

My understanding is that @quasiben started on this work, but ended up pausing to wait for the JupyterLab 1.0.0 release (which this project requires).  Given that that release has occurred (woo!) we might consider pushing on this again.  

@quasiben can you summarize the current state of things.  Also cc'ing @jacobtomlinson, who might find this interesting.

**Comment by quasiben** [comment]
Sure!

1) Create jlab env (v1.0.0 was release 4 days ago)
2) Install jlab 1.0
3) Install [pynvml](https://github.com/gpuopenanalytics/pynvml)
4) Install jupyterlab-bokeh-server (pynvml extensions have already been merged into jupyterlab-bokeh-server) some time ago.  

@jacobtomlinson, I can actively help out tomorrow if you're around and working on this

**Comment by rjzamora** [comment]
> Given that that release has occurred...

Woohoo!   Also, Happy to help @quasiben and @jacobtomlinson with anything they need for this

**Comment by ian-r-rose** [comment]
Glad to see this moving forward -- let me know if you need me to unblock on anything, or if any of you want commit rights here.

**Comment by mrocklin** [comment]
Including link to publcation instructions here: https://github.com/ian-r-rose/jupyterlab-bokeh-server#publishing

**Comment by jacobtomlinson** [comment]
I am definitely interested in this. I'll try and look at it this week.

**Comment by quasiben** [comment]
@jacobtomlinson has built https://github.com/jacobtomlinson/jupyterlab-nvdashboard, should this issue now be closed ?

**Comment by mrocklin** [comment]
Yes
